src/app/models/AudioEventImportFile.ts

```diff
@@ -0,0 +1,68 @@
+import { CollectionIds, DateTimeTimezone, FilePath, Id } from "@interfaces/apiInterfaces";
+import {
+  ANALYSIS_JOB_ITEM,
+  AUDIO_EVENT_IMPORT,
+  TAG,
+} from "@baw-api/ServiceTokens";
+import { bawDateTime, bawPersistAttr, bawSubModelCollection } from "./AttributeDecorators";
+import { AbstractModel } from "./AbstractModel";
+import { hasMany, hasOne } from "./AssociationDecorators";
+import { AnalysisJobItem } from "./AnalysisJobItem";
+import { Tag } from "./Tag";
+import { AudioEventImport } from "./AudioEventImport";
+import { IImportedAudioEvent, ImportedAudioEvent } from "./AudioEventImport/ImportedAudioEvent";
+
+export interface IAudioEventImportFile {
+  id?: Id;
+  fileHash?: string;
+  path?: FilePath;
+  createdAt?: DateTimeTimezone;
+  name?: string;
+  committed?: boolean;
+  importedEvents?: IImportedAudioEvent[];
+
+  analysisJobsItemId?: Id;
+  audioEventImportId?: Id;
+  file?: File;
+  additionalTagIds?: CollectionIds;
+}
+
+export class AudioEventImportFile
+  extends AbstractModel<IAudioEventImportFile>
+  implements IAudioEventImportFile
+{
+  public readonly kind = "audio_event_import_file";
+  public readonly id?: Id;
+  public readonly name?: string;
+  @bawPersistAttr()
+  public readonly fileHash?: string;
+  @bawPersistAttr()
+  public readonly path?: FilePath;
+  @bawDateTime()
+  public readonly createdAt?: DateTimeTimezone;
+  public readonly committed?: boolean;
```

Comment:
maybe add docs: this field is only present immediately after creation (dry or not)